src/olympia/addons/tasks.py

```diff
@@ -417,3 +426,42 @@ def disable_addons(addon_ids, **kw):
     addons.update(status=amo.STATUS_DISABLED, _current_version=None)
     File.objects.filter(version__addon__in=addons).update(status=amo.STATUS_DISABLED)
     index_addons.delay(addon_ids)
+
+
+@task
+@use_primary_db
+def flag_high_hotness_according_to_review_tier():
+    from olympia.reviewers.models import UsageTier
+
+    due_date_generator = get_staggered_review_due_date_generator()
+    usage_tiers = UsageTier.objects.filter(
+        # We don't compute hotness below that, other signals could still apply.
+        lower_adu_threshold__gte=MINIMUM_ADU_FOR_HOTNESS_NONTHEME,
+        # Tiers with no upper adu threshold are special cases with their own
+        # way of flagging add-ons for review (either notable or promoted).
+        upper_adu_threshold__isnull=False,
```

Comment:
In the whole promoted classes. In the docs about this project there is a tier for notable and another for recommended/line. Both these tiers will probably exist in the database, if only for consistency (and later for other features), but both already have their own mechanism to force a review on versions (the `listed_pre_review`/`unlisted_pre_review`/`flag_for_human_review` fields on the promoted class)

Reply:
Can you link to a ticket/doc in the GH issue, or link the jira ticket to the story/epic?

Reply:
Come to think of it though, maybe I can let the flagging happen anyway ? Since those versions are reviewed by humans anyway, they ultimately won't get flagged.

Reply:
Added the signals doc to the issue. The Review Tiers are defined in the appendixes (it's the same tiers we've been discussing for a while with the now defunct DC2 project - I've been avoiding defining them in code in case they change or we don't want to reveal what they are to the outside world).

Reply:
If these tiers are to be used for `Notable` I guess we should be using a `UsageTier` instance for Notable rather than `NOTABLE_ADU_LIMIT_CONFIG_KEY` then?  Or... I suppose it's just not clear to me how this is all going to fit together.

Reply:
Yes, good point, I had that in mind but forgot to file it. It's up now: https://github.com/mozilla/addons-server/issues/20734